auto_submit/test/requests/github_webhook_test.dart

```diff
@@ -12,18 +14,77 @@ void main() {
     late GithubWebhook githubWebhook;
 
     setUp(() {
-      req = Request('POST', Uri.parse('http://localhost/'),
-          headers: {
-            'header1': 'header value1',
+      req = Request('POST', Uri.parse('http://localhost/'), headers: {
+        'header1': 'header value1',
+      }, body: '''{
+      "action": "open",
```

Comment:
This body should be somewhere easily accessible as it's likely we're going to reuse it in a lot of tests. Either move it out of setup or into its own file.

Reply:
+1